src/entry.h

```diff
@@ -264,6 +264,7 @@ class Entry
     QCString     includeName; //!< include name (3 arg of \\class)
     QCString     doc;         //!< documentation block (partly parsed)
     int          docLine;     //!< line number at which the documentation was found
+    int          topLine;     //!< line number at which the page / section definition was found
```

Comment:
In the example you provided `topLine` is always equal to `startLine` (and different from `docLine`) it seems. So is there a case where `topLine` is really needed or can we just use `startLine` instead?

Reply:
For commentscan.l this is true, but in e.g. fortranscanner.l this is not the case (no topline used).
So I was not certain that I could use startLine and decided to use topLine even though this would require some more memory.